ui/views/base.jsx

```diff
@@ -60,7 +61,8 @@ const Layout = ({
       footerLinks={footerLinks}
       phaseBanner={{
         phase: 'beta',
-        content: <span>Your opinions make things better. Share them in <a href="https://www.homeofficesurveys.homeoffice.gov.uk/s/D6XIF5/" target="_blank" rel="noopener noreferrer">this short survey</a>.</span>
+        feedbackUrl: '/feedback',
+        content: phaseBannerSurvey && <span>Your opinions make things better. Share them in <a href="https://www.homeofficesurveys.homeoffice.gov.uk/s/D6XIF5/" target="_blank" rel="noopener noreferrer">this short survey</a>.</span>
```

Comment:
```suggestion
        content: phaseBannerSurvey
```

I'd just pass the content in as a prop from the implementation so it's less of a pain to modify in future.

I'd probably also rename the prop to `phaseBannerContent` while I'm there.

Reply:
yeah, then it needs to be JSX and I can't set it in locals

Reply:
without the markdown dance

Reply:
That sucks.